### ISSUE TITLE
A bug founded when Notepad++ running in Win10

### ISSUE THREAD

**Issue by Lmaj7** [notepad-plus-plus/notepad-plus-plus#2840]
### Description of the Issue

More than one Desktops are available in Win10. When Notepad++ is running and the "Document Map"(Toolbar -> View ->Document Map) is checked, there will be something in whatever which desktop you are using.
Like this:
Picture 1:
<img src="http://i1.piimg.com/4851/f228b51f910f8509.png">
Picture 2:
Pay attention to the right side of the screen.
<img src="http://i1.piimg.com/4851/ca5ec2c35b16e20b.png">

### Steps to Reproduce the Issue

1. Run Notepad++ in Windows10
2. Click on "View" on the toolbar
3. Check "Document Map"
4. Switch to another desktop

**Comment by dail8859** [comment]
Duplicate of #440, #1247, #1317, #2763

**Comment by feinstein** [comment]
Any updates on this bug?

Version 7.5.1 is out, it includes even a spiritual quote...but still, the bug persists....

**Comment by F18** [comment]
Bug still present in Version 7.5.4 on Windows 10. Any updates on this? 

The document map is a fantastic feature, but I'm forced to disable it when using multiple desktops due to this bug.

**Comment by CookiePLMonster** [comment]
Responding so I can track this easier.

**Comment by feinstein** [comment]
1922 issues, I don't think they are going to look at this any time soon :/

**Comment by CookiePLMonster** [comment]
I spent some time investigating it and came to the conclusion this functionality is **terribly broken**. The things you see "leaking" to the other virtual desktop are child dialogs and looks like this is either not permitted or flat out broken. I spent several hours trying to fix this but I still have no ideas. Feels like this approach is wrong to begin with.

**Comment by feinstein** [comment]
I appreciate you spent some of your time into it. Can you think of other ways of achieving this same result? Will them be hard to implement?

**Comment by CookiePLMonster** [comment]
I tried a few approaches and could not come with anything bugless and with equivalent feature level. I am sure there is a way to work it around somehow, but I haven't gotten to that yet.

Frankly, the entire Document Map (and other dockable dialogs) are horrible... they take a few seconds to redraw each time after you drag them etc. - I managed to fix this specific error, but all the other ones (excluding the reported one) still persist. Ehh.
